src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts

```diff
@@ -161,7 +274,9 @@ export function useSwapInputsController({
   });
 
   const formattedOutputNativeValue = useDerivedValue(() => {
-    if (inputMethod.value === 'slider' && percentageToSwap.value === 0 && !inputValues.value.outputNativeValue) return '$0.00';
+    if ((inputMethod.value === 'slider' && percentageToSwap.value === 0) || !inputValues.value.outputNativeValue) {
+      return '$0.00';
```

Comment:
native currency shiz

Reply:
related: https://linear.app/rainbow/issue/APP-1346/switch-tolocalestring-to-support-other-nativecurrencies